Engine/TransactionHandlers/BrokerageTransactionHandler.cs

```diff
@@ -1116,7 +1116,7 @@ public decimal RoundOffOrder(Order order, Security security)
         /// This procedure is needed to meet brokerage precision requirements.
         /// </remarks>
         /// </summary>
-        private void RoundOrderPrices(Order order, Security security)
+        public void RoundOrderPrices(Order order, Security security)
```

Comment:
If setting to `public` just for testing, maybe its worth setting it as protected, inheriting `BrokerageTransactionHandler` and accessing the method through a proxy